js/ui/scheduler/appointments/appointmentCollection.js

```diff
@@ -93,16 +93,16 @@ class SchedulerAppointments extends CollectionWidget {
             }
         };
 
-        const currentAppointment = this._$currentAppointment?.dxResizable('instance');
+        const currentAppointment = this._$currentAppointment;
```

Comment:
Let's rename this variable and leave it as is:

```suggestion
        const currentApptResizable = this._$currentAppointment?.dxResizable('instance');
```

Reply:
I had to remove this part because during dragging, when pressing escape, the _supportedKeys method work out and the this._$currentAppointment instance are changed.

Reply:
Ok, got it